plugins/plugin_word_respond.py

```diff
@@ -0,0 +1,43 @@
+from __future__ import unicode_literals
+
+import re
+from collections import Counter
+
+from rtmbot.core import Plugin
+
+
+class PluginWordRespond(Plugin):
+    def __init__(self, slack_client=None, plugin_config=None):
+        # because of the way plugins are called we must explicitly pass the
+        # arguments to the super
+        super(PluginWordRespond, self).__init__(slack_client, plugin_config)
+        self.words = self.load_word_list("words.txt")
+
+    def process_message(self, data):
+        print(data)
+        if (data['channel'].startswith("C") and data['type'] == 'message'):
+            word_counter = self.check_for_flag_words(data['text'])
+            if word_counter:
+                resp = self.slack_client.api_call('im.open', user=data['user'])
+                if resp['ok']:
+                    self.outputs.append([
+                        resp['channel']['id'],
+                        'I counted you saying the following words this many '
+                        'times: \n{}'.format(word_counter)
+                    ])
+
+    def check_for_flag_words(self, message):
```

Comment:
I'd probably separate this out into it's own file and just call it from this plugin. This will make it easier in the future when the method becomes more complicated (if it becomes more complicated)